platforms/platforms-lookup.txt

```diff
@@ -0,0 +1,6 @@
+# Left column is the directory name under ./platforms
+# Right column is the expected result of `hostname --fqdn`
+ls6      ls6
+penguin  penguin
+mike     mike
+hatteras compute-[0-9][0-9]?-[0-9][0-9]?.cluster 
```

Comment:
It is good that the hatteras lookup will only work in an `sinteractive` session because `sinteractive` is required but what if an Operator does not know that and the `$HOSTNAME` is not recognized? Perhaps a 2nd check to see if the $HOSTNAME is `ht1.cluster` and a warning message that this is not supported?